docs/devices-api/getting-started/connect-widget.mdx

```diff
@@ -409,6 +409,22 @@ For a comprehensive list of available colors [visit here](https://www.w3schools.
   src="https://connect.metriport.com/?token=demo&customColor=orange"
 ></iframe>
 
+### redirectUrl
+
+After users successfully connect a provider, they can be redirected to a URL of your choosing.<br/>
+
+```typescript
+`https://connect.metriport.com/?token=${connectToken}&redirectUrl=https://my-app.com/good-news`;
```

Comment:
don't we need to encode the redirect URL before passing it as a param? ie https://www.tutorialspoint.com/how-to-encode-and-decode-url-in-typescript

so this param would be: `https%3A//my-app.com/less-good-news`

would also add a warning for the devs so they're aware